code/datums/components/mood.dm

```diff
@@ -214,16 +214,16 @@
 	var/mob/living/master = parent
 	switch(sanity)
 		if(SANITY_INSANE to SANITY_CRAZY)
-			setInsanityEffect(MAJOR_INSANITY_PEN)
```

Comment:
Nerf the define values but don't remove the effects outright for major or minor insanity.

Reply:
Done